test/e2e/utils/common.js

```diff
@@ -288,10 +288,10 @@ function completeVaBenefits(client, data, onlyRequiredFields) {
 }
 
 function completeFinancialDisclosure(client, data, onlyRequiredFields) {
-  client.click('input[name="understandsFinancialDisclosure"]');
+  client.click('input[name="understandsFinancialDisclosure-1"]');
```

Comment:
I'm torn on this change. On the one hand I feel like we should always click `-1`, because otherwise we can't test the requirements of the spouse/dependent/income/expense pages, but those pages aren't required if they click `-0`. I suppose those requirements are being tested in test 01-required-and-optional?